src/pages/Wishlist/Wishlist.jsx

```diff
@@ -1,20 +1,41 @@
-import React,{useState,useEffect} from 'react'
-import "./Wishlist.css"
-import { useWishlist } from '../../Context/wishlist-context';
-import getDiscount from '../../components/Discount/Discount';
-import { useAuth } from '../../Context/auth-Context';
+import React, { useState, useEffect } from "react";
+import "./Wishlist.css";
+import { useWishlist } from "../../Context/wishlist-context";
+import getDiscount from "../../components/Discount/Discount";
+import { useAuth } from "../../Context/auth-Context";
 import axios from "axios";
-import ProductCard from '../../components/ProductCard/ProductCard';
-import WishlistCard from '../../components/WishlistCard/WishlistCard';
-
+import ProductCard from "../../components/ProductCard/ProductCard";
```

Comment:
add an index.js file in your components folder and import your components from there